templates/search/blocks/search_filters.html

```diff
@@ -45,16 +45,16 @@ <h3 class="tna-heading sr-only">Edit filters</h3>
 
             {% if form.group.value == 'tna' or form.group.value == 'digitised' or form.group.value == 'nonTna' %}
                 <div class="search-filters__accordion-section">
-                    <fieldset>
+                    <fieldset role="group">
                         <legend>
                             <h4 class="tna-heading search-filters__accordion-section-heading" id="record_covering_date">Dates</h4>
                         </legend>
 
                         <span class="example-text">For example, 27 3 2007</span>
                         <div>{{ form.covering_date_from.label_tag }}</div>
 
-                        <div class="search-filters__creation-date-section">
-                            
+                        <div id="from-date">
+                            <fieldset role="group" aria-describedby="from-date">
```

Comment:
Sorry - I don't think I described this very well!

The `aria-describedby="from-date"` needs to be added to the `<fieldset>` on line 48 and then remove these nested `<fieldset>`s.

I'm taking this advice from https://design-system.service.gov.uk/components/date-input/

Reply:
I've now removed the nested `<fieldset>`s and adjusted the `aria-describedby` elements